manifests/qpid_client.pp

```diff
@@ -39,16 +40,25 @@
 
   if $deploy {
 
+    file { $certs::pulp_pki_dir:
```

Comment:
I like explicit ownership and permissions. Otherwise things are affected by umask and such.